src/DataViews/LiveFunctionsDataViewTest.cpp

```diff
@@ -847,13 +853,15 @@ TEST_F(LiveFunctionsDataViewTest, ColumnSortingShowsRightResults) {
 }
 
 TEST_F(LiveFunctionsDataViewTest, OnDataChangeResetsHistogram) {
-  EXPECT_CALL(app_, ShowHistogram(nullptr, "", orbit_client_data::kInvalidScopeId)).Times(1);
+  std::optional<ScopeId> nullopt = std::nullopt;
+  EXPECT_CALL(app_, ShowHistogram(nullptr, "", nullopt)).Times(1);
```

Comment:
Is there a reason for the local variable? Do you need an l-value here?

Reply:
(If you want to be explicit about the type I would propose to use `std::optional<ScopeId>{}` as the argument.)

Reply:
Maybe problems with type inference? For me an alternative to using `std::optional<ScopeId>{}` would be to call the variable `nullopt_scope_id`.

Reply:
Yeah, it's type inference issue. Let's go with `std::optional<ScopeId>{}`. Thank you.